src/main/flight/pid.c

```diff
@@ -778,7 +812,19 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
 
     // Dynamic i component,
     // gradually scale back integration when above windup point
-    const float dynCi = MIN((1.0f - motorMixRange) * ITermWindupPointInv, 1.0f) * dT * itermAccelerator;
+    
+    if (antiGravityNew && antiGravityEnabled) {
+        // Calculate new smooth anti-gravity effect where I accumulates faster with faster throttle movement
+        // antiGravityThrottleHpf typically reaches max of 0.1 for fast throttle changes
+        // AG Gain of 5000 results in itermAccelerateFactor of 5 at antiGravityThrottleHpf of 0.1
+        // if throttle movement is zero, antiGravityThrottleHpf is zero, I accumulates normally
+        itermAccelerator = (1 + (fabsf(antiGravityThrottleHpf) * 0.01f * (itermAcceleratorGain - 1000)));
```

Comment:
The outermost parentheses are surely not needed. Parens around `fabsf() * .. * ..)` also don't make it any more clear IMO.